.github/workflows/test-sarif.yaml

```diff
@@ -10,10 +10,12 @@ on:
   # See info here https://docs.github.com/en/rest/reference/code-scanning#upload-an-analysis-as-sarif-data
   # Although not documented officially, `pull_request` event type will make `github.ref` in PR format.
   # See https://frontside.com/blog/2020-05-26-github-actions-pull_request/#how-does-pull_request-affect-actionscheckout
-  - pull_request
+  pull_request:
   # `push` event type should also be turned on in order for GitHub to be able to figure out diff between alerts in
   # source and target branches of PR and display it in Actions | Code scanning results.
```

Comment:
Please also make sure to update this comment.

Reply:
Updated comment and instead of running only on `main` ignore @dependabot branches